docs/attention_version_agnostic.txt

```diff
@@ -0,0 +1,5 @@
+.. attention::
+   **This document is version–agnostic.
+   For the most accurate, up–to–date information,
+   make sure that you're reading
+   the latest development version of it.**
```

Comment:
A simple way to [handle version agnostic docs](https://github.com/rtfd/readthedocs.org/issues/1493).